.circleci/config.yml

```diff
@@ -12,6 +12,9 @@ workflows:
       - frontend_tests:
           requires:
             - setup
+      - frontend_test_coverage:
+          requires:
+            - setup
```

Comment:
Change this so that it requires `frontend_tests` instead. We want to run this only after we know the frontend tests pass.

Reply:
Done.